src/server/routes/entity/entity.tsx

```diff
@@ -264,14 +265,15 @@ function _createNote(orm, content, editorID, revision, transacting) {
 	return null;
 }
 
-export function addNoteToRevision(req: PassportRequest, res: $Response) {
+export async function addNoteToRevision(req: PassportRequest, res: $Response) {
 	const {orm}: {orm?: any} = req.app.locals;
 	const {Revision, bookshelf} = orm;
 	const editorJSON = req.session.passport.user;
 	const revision = Revision.forge({id: req.params.id});
 	const {body}: {body: any} = req;
-	const revisionNotePromise = bookshelf.transaction(
-		(transacting) => _createNote(
+
+	const revisionNotePromise = await bookshelf.transaction(
```

Comment:
(Same comment as above to rename this variable to `revisionNote`)

We don't want to use `sendPromiseResult` here as it is mainly a utility for awaiting a promise safely and sending the result.
But since you're already `await`ing the promise here, you should wrap it in a try catch block and directly return a response like so:
`return res.status(200).send(revisionNote);`

In the catch block you can `return next(err);`